src/sentry/static/sentry/app/stores/hookStore.tsx

```diff
@@ -2,7 +2,7 @@ import Reflux from 'reflux';
 import _ from 'lodash';
 import * as Sentry from '@sentry/browser';
 
-const validHookNames = new Set([
+const validHookNames: Set<string> = new Set([
```

Comment:
If anything i feel like _this_ would be an enum.

Reply:
@EvanPurkhiser 

I found a much cooler approach than enums using the `as const` syntax: https://github.com/getsentry/sentry/pull/14160/commits/dfb9b9874b19d3d26a318bcd9a8caabafc79fbc4

From https://stackoverflow.com/a/45257357/412627